src/status_im/transport/message/transit.cljs

```diff
@@ -44,14 +44,36 @@
   (rep [this {:keys [name profile-image address fcm-token]}]
     #js [name profile-image address fcm-token]))
 
+;; It's necessary to support old clients understanding only older, needlesy complex and verbose command content
```

Comment:
Thanks for letting us know you didn't like the previous implementation, but it would be more useful to add which version was the last one to run that code, so we can ditch the code once we are sure that no clients are using it :)

Reply:
Sure, I will add it to the comment, but I don't think it's particularly important as it's version newer then `0.9.20`, so we have to support it (indefinitely ?) according to last team discussion about version compatibility.
Neither is my rant, but you would have to see the old payload to understand my frustration :))

Reply:
I'd rather not see it if it prompted such a bad reaction :) 
I would still add the version number, as it is also for external devs, who might not care about supporting previous versions, and also you never know if we are going to revisit those policies at some point ;)

Reply:
True, I removed the unnecessary emotions and added facts (exact version where command `content` was upgraded to newer, sparser version)